swan-pipeline/src/main/java/de/fraunhofer/iem/swan/features/code/IsMethodConstructorFeature.java

```diff
@@ -0,0 +1,43 @@
+package de.fraunhofer.iem.swan.features.code;
+
+import de.fraunhofer.iem.swan.data.Category;
+import de.fraunhofer.iem.swan.data.Method;
+import de.fraunhofer.iem.swan.features.code.type.WeightedFeature;
+
+import java.util.ArrayList;
+import java.util.Set;
+
+public class IsMethodConstructorFeature extends WeightedFeature implements IFeatureNew {
```

Comment:
Maybe this feature could be combined with a feature to create a method type feature: for example, construction, setter, getter, etc.